### PR TITLE
Move motivation, goals and members to website

### PR DESCRIPTION
**Pull request by infinisil**
And update the text here to reflect that

See https://github.com/NixOS/nixos-homepage/pull/907

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -115,7 +74,7 @@ The process of joining the team is as follows:
 - A discussion will take place as to whether you're accepted as a new team member
 - The current team members will reach consensus on whether to accept the request and record that decision in the public [team log](https://github.com/nixpkgs-architecture/team-log) document, including all relevant arguments
 - If accepted, [@infinisil:matrix.org](https://matrix.to/#/@infinisil:matrix.org) will take these steps for adding you to the team:
-  - Updating the above [team list](#current-team) list with your entry and all relevant details
+  - Updating the team list [here](https://nixos.org/community/teams/nixpkgs-architecture.html) with your entry and all relevant details
```
Comment (roberth) [inline profile/README.md]:
```suggestion
  - Updating the team list [here](https://github.com/NixOS/nixos-homepage/blob/master/community/teams/nixpkgs-architecture.tt) with your entry and all relevant details
```

File: profile/README.md [hunk 2]
```diff
@@ -137,7 +96,7 @@ If there is a hint that a team member doesn't fulfil the requirements anymore bu
 
 The process for voluntarily or involuntarily removing a team member is done by [@infinisil:matrix.org](https://matrix.to/#/@infinisil:matrix.org) and goes as follows:
 - An removal entry is added to the public [team log](https://github.com/nixpkgs-architecture/team-log) including all relevant arguments for removal
-- Remove the respective team member from the above [team list](#current-team)
+- Remove the respective team member from the team list [here](https://nixos.org/community/teams/nixpkgs-architecture.html)
```
Comment (roberth) [inline profile/README.md]:
```suggestion
- Remove the respective team member from the team list [here](https://github.com/NixOS/nixos-homepage/blob/master/community/teams/nixpkgs-architecture.tt)
```

**Comment by roberth** [comment]
The nixos.org page has some underscores. Is that intentional?

**Comment by infinisil** [comment]
@roberth Was a mistake, fixing it in https://github.com/NixOS/nixos-homepage/pull/919

**Pull request merged**